pkg/mutation/system.go

```diff
@@ -140,12 +181,16 @@ func logAppliedMutations(message string, obj *unstructured.Unstructured, allAppl
 	}
 	if len(iterations) > 0 {
 		logDetails := []interface{}{}
+		logDetails = append(logDetails, "Mutation Id", mutationUUID)
 		logDetails = append(logDetails, logging.EventType, logging.MutationApplied)
 		logDetails = append(logDetails, logging.ResourceGroup, obj.GroupVersionKind().Group)
 		logDetails = append(logDetails, logging.ResourceKind, obj.GroupVersionKind().Kind)
 		logDetails = append(logDetails, logging.ResourceAPIVersion, obj.GroupVersionKind().Version)
 		logDetails = append(logDetails, logging.ResourceNamespace, obj.GetNamespace())
 		logDetails = append(logDetails, logging.ResourceName, obj.GetName())
+		if timestamp != nil {
```

Comment:
Don't log lines normally have a timestamp embedded by default? Given we have the UUID, are we expecting useful debug information from the timestamp?

Reply:
Right.
This was the unique timestamp identifying the mutation, now not needed as we have the id